src/scenes/Authentication/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,40 @@
+import { ApolloError } from '@apollo/client';
+import { FORM_ERROR } from 'final-form';
+import React from 'react';
+import { Except } from 'type-fest';
+import { useForgotPasswordMutation } from '../../../api';
+import {
+  ForgotPasswordForm,
+  ForgotPasswordFormProps as Props,
+} from './ForgotPasswordForm';
+
+export const ForgotPassword = (props: Except<Props, 'onSubmit'>) => {
```

Comment:
We also need to have a success state where we show the next step instead of the form. Sorry this wasn't clear in the ticket.

A separate `ForgotPasswordSuccess` component makes sense. 
For now it can just have a go to login button